Dockerfile

```diff
@@ -0,0 +1,55 @@
+# Use Ubuntu 16.04 LTS
+FROM nvidia/cuda:9.1-runtime-ubuntu16.04
+
+# Prepare environment
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+                    curl \
+                    bzip2 \
+                    ca-certificates \
+                    build-essential \
+                    autoconf \
+                    libtool \
+                    pkg-config \
+                    wget \
+                    libboost-all-dev \
+                    unzip \
+                    libgl1-mesa-dev \
+                    libglu1-mesa-dev \
+                    freeglut3-dev \
+                    mesa-utils \
+                    g++ \
+                    jq \
+                    tar \
+                    zip \
+                    gcc \
+                    make \
+                    python \
+                    zlib1g-dev \
+                    imagemagick \
+                    software-properties-common \
+                    git && \
+    curl -sL https://deb.nodesource.com/setup_10.x | bash - && \
```

Comment:
pretty sure we don't need node

Reply:
@mattcieslak do you think it's worth trimming the package list a bit?

Reply:
many of these can go. They came with on a copy/paste from another dockerfile